src/main/java/Duke.java

```diff
@@ -1,10 +1,174 @@
+        }
+        Duke.exit();
+    }
+
+    public static void greet() {
+        Duke.printLine();
+        System.out.println("\t" + "Hello! I'm Duke\n" +
+                "\t" + "What can I do for you?");
+        Duke.printLine();
+    }
+
+    public static void commandAgain(String command) {
+        Duke.printLine();
+        switch(command) {
+        case "todo":
+            System.out.println("\t🙁 OOPS!!! The description of a todo cannot be empty.");
+            break;
+        case "deadline":
+            System.out.println("\t🙁 OOPS!!! The description of a deadline cannot be empty.");
+            break;
+        case "event":
+            System.out.println("\t🙁 OOPS!!! The description of a event cannot be empty.");
+            break;
+        default:
+            System.out.println("\t🙁 OOPS!!! I'm sorry, but I don't know what that means :-(");
+            break;
+        }
+        System.out.println("\tPlease give a valid command!");
+        Duke.printLine();
+    }
+
+    public static void displayList(ArrayList<Task> list) {
+        Duke.printLine();
+        System.out.println("\tHere are the tasks in your list:");
+        for(int i=1; i<=list.size(); i++) {
+            System.out.println("\t" + i + ". " + list.get(i-1).toString());
+        }
+        Duke.printLine();
+    }
+
+    public static void printAdd(Task task) {
+        Duke.printLine();
+        System.out.println("\tGot it. I've added this task: ");
+        System.out.println("\t   " + task.toString());
+        System.out.println("\tNow you have " + (list.size()) + " tasks in the list. ");
+        Duke.printLine();
+    }
+
+    public static void addTodo (String command) throws DukeException {
+        if(command.equals("todo")) {
+            throw new DukeException();
+        }
+        list.add(new ToDo(command.substring(5)));
+        Duke.printAdd(list.get(list.size()-1));
+    }
+
+    public static void addDeadline(String command) throws DukeException{
+        if(command.equals("deadline")) {
+            throw new DukeException();
+        }
+        int separatePosition = command.indexOf('/');
+        String description = command.substring(SKIP_DEADLINE_NUMBER, separatePosition-1);
+        String byTime = command.substring(separatePosition+SKIP_SEPARATION_NUMBER);
+        list.add(new Deadline(description, byTime));
+        Duke.printAdd(list.get(list.size()-1));
+    }
+
+    public static void addEvent(String command) throws DukeException{
+        if(command.equals("event")) {
+            throw new DukeException();
+        }
+        int separatePosition = command.indexOf('/');
+        String description = command.substring(SKIP_EVENT_NUMBER, separatePosition-1);
+        String atTime = command.substring(separatePosition+SKIP_SEPARATION_NUMBER);
+        list.add(new Event(description, atTime));
+        Duke.printAdd(list.get(list.size()-1));
+    }
+
+    public static void done(ArrayList<Task> list, String command) {
```

Comment:
"done" is very vague. Consider being more descriptive in your name by stating what is being done